package.json

```diff
@@ -128,6 +133,7 @@
 		"husky": "3.0.5",
 		"inquirer": "6.3.1",
 		"is-equal-shallow": "0.1.3",
+		"jest": "24.1.0",
```

Comment:
It's a bit concerning that older version of Jest is set here while the project uses `24.9.0`:
https://github.com/WordPress/gutenberg/blob/feed14ac384266b0ffe0740e1e99f0eb7d0c30f7/packages/scripts/package.json#L52

Can we fix it for both `jest` and `babel-jest`?